src/userEvent/touchEvents.ts

```diff
@@ -0,0 +1,53 @@
+import { ReactTestInstance } from 'react-test-renderer';
+import act from '../act';
+import { getHostParent } from '../helpers/component-tree';
+import { isHostElementPointerEventEnabled } from '../helpers/isHostElementPointerEventEnabled';
+
+const defaultPressEvent = {
+  persist: jest.fn(),
+  nativeEvent: {
+    timestamp: new Date().getTime(),
+  },
+  currentTarget: { measure: jest.fn() },
+};
+
+export type PressOptions = {
+  pressDuration: number;
```

Comment:
Q: What are the use cases for `pressDuration` prop?

Reply:
The main use case is to simulate a long press. If press duration is greater or equal than the long press duration offset then it will call onLongPress and not onPress. You can look at the test cases for pressDuration, it should make things clearer

Reply:
Hmmm, exposing `pressDuration` is looking like quite a low-level implementation detail. To use it properly it would require user to know the default press duration of `Pressable`. I think a simpler API would be to expose two events:

```
press: () => void,
longPress: ({ duration = 500 }) => void
```

As for regular presses, which are far more popular than long press, the duration does not matter that much, and for `longPresses` most of the users would probably stick with the default `pressDuration` value. If they tweaked it, then they would also have option to apply the theaked value in test.

Reply:
I don't agree with press duration being an implementation detail, it's directly related to the user action. In my opinion longPress will be interpreted like it triggers my onLongPress prop so I think it's more linked to implementation details. So I think it's a good addition to be able to change the duration, having an additional longPress api with a different default may be a good idea I'm not too sure on this